pkg/registries/ecr/ecr.go

```diff
@@ -82,63 +73,9 @@ func sanitizeConfiguration(ecr *storage.ECRConfig) error {
 	return errorList.ToError()
 }
 
-func (e *ecr) refreshDockerClient() error {
-	if e.expiryTime.After(time.Now()) {
-		return nil
-	}
-	if e.integration.GetEcr().GetAuthorizationData() != nil {
-		// This integration has static authorization data, and we never refresh the
-		// tokens in central, rather we wait for sensor to update them.
-		return errors.New("failed to refresh the auto-generated integration credentials")
-	}
```

Comment:
Is this no longer needed?

Reply:
The idea is to have the transport handle the authorization data. There are two options:
* `GetAuthorizationData() != nil`: Static credentials exist in the database. In this case we create the standard transport that uses static credentials, no refresh ever happens in Central. We rely on the credentials being valid and handled by Sensor.
* `GetAuthorizationData() == nil`: We create a transport with access to the AWS client. The client is used to fetch new credentials when they expire. This happens in the transport and we longer need a `refreshDockerClient` on the registry level.